jsk_perception/sample/sample_fcn_object_segmentation.launch

```diff
@@ -16,6 +16,24 @@
     <arg name="GPU" value="$(arg GPU)" />
```

Comment:
I think it is better if you put separated launch file named `sample_<node_name>.launch`. What do you think?